docs/editing-the-docs.md

```diff
@@ -0,0 +1,34 @@
+# Editing the Docs
+A quick guide for editing and updating our documentation.
+
+Most text can easily be edited with your favourite text editor, or directly in GitHub itself. There's really nothing special here.
+
+#### Editing Diagrams
+Editing diagrams is the tricky part about our documentation. All our diagrams live in the [images](images/) folder. These are all diagrams created by [draw.io](https://www.draw.io/?mode=github). Specifically, this is done with the "GitHub mode" of [draw.io](https://www.draw.io/?mode=github). This allows diagrams or images to be saved directly in a git repository, but still edited with [draw.io](https://www.draw.io/?mode=github) when needed.
```

Comment:
note that draw.io can be run offline :wink:

Reply:
I think that's if you download it? Not sure how that plays with the "GitHub mode", but good to know.